### PR TITLE
Create new TF using Ubuntu base image for service tunnel

### PR DESCRIPTION
**Pull request by tdesikan**

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -132,7 +121,7 @@ variable "custom_user_data" {
 variable "redirect_http_to_https" {
   type        = bool
   description = "If true, requests to the AccessTier on port 80 will be redirected to port 443"
-  default     = false
+  default     = true
```
Comment (themethod) [inline variables.tf:124]: what's the reason to make this default to true?

Comment (themethod) [reply variables.tf:124]: the README should be updated to remove references to amazon Linux and custom ami (ami_id)

Comment (themethod) [reply variables.tf:124]: and `default_ami_name`

Comment (tdesikan) [reply variables.tf:124]: http->https is true in our CF template: https://github.com/banyansecurity/installer/blob/master/cloudformation/banyan-service-tunnel-ubuntu.json#L371

 i'd like to keep it consistent everywhere.

Comment (themethod) [reply variables.tf:124]: ok, that makes sense. just update README.md then and it's good to go.

**Pull request merged**